### PR TITLE
Document stream writer `typing`, `to_device`, `account_data`, `receipts`, and `presence` can only be used on a single worker.

### PR DESCRIPTION
**Pull request by cremesk**

### CODE REVIEW

**Comment by cremesk** [comment]
> Can you [sign-off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off) on your changes please?

i hope I made it.

**Comment by clokep** [comment]
Sign-off looks good, thank you! 👍

File: changelog.d/12187.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Remove unused variables.
```
Comment (clokep) [inline changelog.d/12187.misc:1]: There seems to be some unrelated changes in this PR.

File: changelog.d/12196.doc [hunk 2]
```diff
@@ -0,0 +1 @@
+Document stream writer `to_device`, `account_data`, `receipts`, and `presence` can only be used on a single worker.
```
Comment (clokep) [inline changelog.d/12196.doc:1]:
```suggestion
Document that the`to_device`, `account_data`, `receipts`, and `presence` stream writers can only be used on a single worker.
```

Please add "Contributed by `<your name>`." if you'd like to be credited!

File: docs/workers.md [hunk 3]
```diff
@@ -384,38 +384,38 @@ stream_writers:
 
 ##### The `typing` stream
 
-The following endpoints should be routed directly to the workers configured as
-stream writers for the `typing` stream:
+The following endpoints should be routed directly to a single worker configured
+as stream writers for the `typing` stream:
```
Comment (clokep) [inline docs/workers.md:388]: Would it make more sense to modify the paragraph under `Stream writers` to say that all stream writers can only go to a single instance, except where documented?

Comment (cremesk) [reply docs/workers.md:388]: yes, that would probably be the better option. i change it.

**Review by clokep — APPROVED** [review]
I moved around a sentence or two, but I think this looks great! Thank you! 🎉

**Comment by cremesk** [comment]
> I moved around a sentence or two, but I think this looks great! Thank you! tada

With pleasure! I am glad that I could help :blush:
